streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -25,7 +25,7 @@
 import org.apache.kafka.clients.consumer.KafkaConsumer;
 import org.apache.kafka.clients.producer.KafkaProducer;
 import org.apache.kafka.clients.producer.ProducerConfig;
-import org.apache.kafka.common.KafkaException;
+import org.apache.kafka.common.errors.TimeoutException;
```

Comment:
I think you need to move this line before line 47 to have the imports lexicographically sorted.